ci/docker/detect-secrets/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM python:3.11.0-alpine3.16@sha256:610aac6d972a31b2d4174a8b260e2cdf69de2d73cf03f4131ca24e61157f34d6
+
+RUN apk update && apk upgrade
+RUN apk add bash git
```

Comment:
That script will work as plain sh, you could just change the #! to `#!/bin/sh` since it's all POSIX compatible and then you won't need to install bash

Reply:
good to know, I'd like to keep it though even just to help with future debugging / dev efforts if we revisit this action (and it's only a 1MB difference!)